.github/workflows/ci.yml

```diff
@@ -17,10 +17,10 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11]
```

Comment:
Here you should use 3.10.8 or the zero will get eaten.

Reply:
Ok, Thanks for the info.

Reply:
This is now correct, resolving this comment.